packages/compartment-mapper/NEWS.md

```diff
@@ -1,5 +1,20 @@
 User-visible changes to `@endo/compartment-mapper`:
 
+# Next release
+
+- Archives will now automatically treat any module specifier that appears
+  to start with a URL scheme (like `node:fs`) and these "exits" will no
+  longer need to be marked in the `"modules"` option.
```

Comment:
Grammar? I'm having a hard time parsing this sentence.

Reply:
This should be more clear now.